src/components/views/rooms/ForwardMessage.js

```diff
@@ -45,7 +44,6 @@ export default createReactClass({
     componentWillUnmount: function() {
         dis.dispatch({
             action: 'panel_disable',
```

Comment:
is this action still used? Or you wanted to avoid expanding the scope of your PR?

Reply:
As in it has other listeners, I deemed it outside of scope to see whether left panel/middle panel disabling is valid/desired/functional.